### PR TITLE
CAN-135: Manage Profile: Implement Google Places API for address

### PR DESCRIPTION
**Pull request by vikasranatalentelgia**
Manage Profile: Implement Google Places API for address

### CODE REVIEW

File: src/components/ComponentPages/ProfileInfo/index.tsx [hunk 1]
```diff
@@ -124,6 +128,65 @@ const ProfileInfo = () => {
...
+
+  const handleAddressSelect = async (address, placeId) => {
+    setAddress(address)
+    const results = await geocodeByAddress(address);
+    const [place] = await geocodeByPlaceId(placeId);
+    const { long_name: postalCode = '' } =
+      place.address_components.find(c => c.types.includes('postal_code')) || {};
+    var city = ""
+    var country = ""
+    var address2 = ""
```
Comment (akuks) [inline src/components/ComponentPages/ProfileInfo/index.tsx]: How about this
` let city = country = address2 = ""`

File: src/components/ComponentPages/ProfileInfo/index.tsx [hunk 2]
```diff
@@ -124,6 +128,65 @@ const ProfileInfo = () => {
...
+      }
+    }
+    address2 = address2.replace(/^,|,$/g, '');
+
+    form.setFieldsValue({ ["address_2"]: address2 })
+    form.setFieldsValue({ ["postal_code"]: postalCode })
+    form.setFieldsValue({ ["city"]: city })
+    form.setFieldsValue({ ["state"]: state })
+    form.setFieldsValue({ ["country"]: country })
+  }
```
Comment (akuks) [inline src/components/ComponentPages/ProfileInfo/index.tsx]: Probably we need to break this. This doesn't looks good

```javascript

for (const component of results[0].address_components) {
      const componentType = component.types[0];
      
      address2 = getAddress(componentType, address2, component);

 .....
......
.....
}


// Create a function similar to this
function getAddress( type, address, component ) {
    if ( type === "neighbourhood" ) {
        return component.long_name
    }
    else if ( type.match( /^neighborhood$|^sublocality_level_2$|^sublocality_level_1$/ ) ) {
        return address + ", " + component.long_name
    }
    else { 
        return 0
    }
  }

```

Comment (akuks) [reply src/components/ComponentPages/ProfileInfo/index.tsx]: `city`, `state` and `country` are state forward

**Review by akuks — CHANGES_REQUESTED** [review]
Please go through the comments

File: src/components/ComponentPages/ProfileInfo/index.tsx (hunk 1 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ProfileInfo/index.tsx]: done

File: src/components/ComponentPages/ProfileInfo/index.tsx (hunk 2 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ProfileInfo/index.tsx]: done

**Review by akuks — APPROVED** [review]

**Pull request merged**